### PR TITLE
Adaptions to documentation errors

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
This PR is to solve some issues with the docs build and also fix documentation. 

The EventListMetaData was the only metadata object which was exposed in the API. An important question is whether we want to expose these or not?

### CODE REVIEW

**Review by bkhelifi — DISMISSED** [review]
Thanks a lot @Astro-Kirsty, well seen and fixed ;)

**Review by AtreyeeS — DISMISSED** [review]
Thanks @Astro-Kirsty for this cleanup!

File: gammapy/data/__init__.py [hunk 1]
```diff
@@ -16,7 +16,6 @@
 __all__ = [
     "DataStore",
     "EventList",
-    "EventListMetaData",
```
Comment (bkhelifi) [inline gammapy/data/__init__.py:19]: Why is it removed?

File: gammapy/modeling/models/spatial.py [hunk 2]
```diff
@@ -1323,10 +1323,7 @@ def __init__(
                 data_sum = map.data.sum(axis=(1, 2)).reshape((-1, 1, 1))
 
             data = np.divide(
-                map.data.astype(float),
-                data_sum,
-                out=np.zeros_like(map.data, dtype=float),
-                where=data_sum != 0,
+                map.data, data_sum, out=np.zeros_like(map.data), where=data_sum != 0
```
Comment (bkhelifi) [inline gammapy/modeling/models/spatial.py:1326]: Why the float conversion is removed?

**Review by bkhelifi — COMMENTED** [review]
Thanks a lot @Astro-Kirsty ... so many small details that are corrected ;)
I have 2 questions in line

File: gammapy/data/__init__.py (hunk 1 above)
Comment (Astro-Kirsty) [reply gammapy/data/__init__.py:19]: Rebase gone wrong, fixing now!

File: gammapy/modeling/models/spatial.py (hunk 2 above)
Comment (Astro-Kirsty) [reply gammapy/modeling/models/spatial.py:1326]: Rebase gone wrong!

**Review by bkhelifi — DISMISSED** [review]
All good! Thank you

**Comment by Astro-Kirsty** [comment]
Do we want to expose the rest of the metadata classes/function in the API?

**Review by registerrier — APPROVED** [review]
Thanks @Astro-Kirsty ! Looks good.

**Pull request merged**